### ISSUE TITLE
Publisher should not see status 3 bricks in the public catalogue

### ISSUE THREAD

**Issue by scholar6admin** [Scholar-6/brillder#4607]

**Comment by ivanromakh** [comment]
I will change the rules.

**Comment by ivanromakh** [comment]
I don`t think this is possible. they all have status 4. I don`t see any brick in dev with status 3 or less.

**Comment by ivanromakh** [comment]
for catalog we use separate request to get by /bricks/byStatus/4

**Comment by s-steddy** [comment]
@ivanromakh (hi again!) It looks like this was fixed too. I am a publisher and can't see bricks pending publication as I scroll the catalogue. Do I need to try anything else?
